org.opent2t.sample.binaryswitch.superpopular/com.wink.binaryswitch/js/tests/test.js

```diff
@@ -1,13 +1,10 @@
 var test = require('ava');
 var OpenT2T = require('opent2t').OpenT2T;
-var config = require('./testConfig');
-
-console.log("Config:");
-console.log(JSON.stringify(config, null, 2));
-
 var translatorPath = require('path').join(__dirname, '..');
+var runBinarySwitchTests = require('opent2t-device-binaryswitch/binarySwitchTests');
+var config = require('./testConfig');
 var hubPath = require('path').join(__dirname, '../../../../org.opent2t.sample.hub.superpopular/com.wink.hub/js');
-var translator = undefined;
+var deviceId = "F85B0738-6EC0-4A8B-A95A-503B6F2CA0D8";

```

Comment:
>F85B0738-6EC0-4A8B-A95A-503B6F2CA0D8 [](start = 16, length = 36)

Is there a reason why we are hard coding the deviceID instead of getting it from platfromInfo during the translator setup?